metadata-db-app/test/cmr/metadata_db/test/data/oracle/concepts.clj

```diff
@@ -38,7 +39,7 @@
 ;; This test is commented out until CMR-1303 is resolved
 #_(deftest db-result->concept-map-test
```

Comment:
This test should probably be deleted if it hasn't been re-enabled since CMR-1303 was written.